lib/archethic_web/live/network_stats.ex

```diff
@@ -0,0 +1,66 @@
+defmodule ArchEthicWeb.NetworkStats do
+  @moduledoc false
+
+  use Phoenix.LiveView
+  use Phoenix.HTML
+
+  def mount(_params, _session, socket) do
+    if connected?(socket) do
+      ArchEthic.Metrics.MetricClient.monitor()
+      :timer.send_interval(5_000, self(), :update)
+    end
+
+    {:ok, socket}
+  end
+
+  def handle_info(:update, socket) do
```

Comment:
Then by pushing, you should get the points in the message avoiding a GenServer.call